Scripts/script-TaniumDeployAction.yml

```diff
@@ -104,3 +105,4 @@ dependson:
   must:
   - tn-deploy-package
 timeout: 0s
+releaseNotes: Deprecated script
```

Comment:
please write the command that replace this script.
Deprecated script, use `command name` instead.

Reply:
The command is tn-deploy-package